config.dev.toml

```diff
@@ -211,3 +211,14 @@ TraceExporter = ""
         # S3 Bucket to use for storage
         # Env override: ATHENS_S3_BUCKET_NAME
         Bucket = "MY_S3_BUCKET_NAME"
+   [Storage.AzureBlob]
+        # Storage Account name for Azure Blob
+        AccountName = "AZURE_BLOB_ACCOUNT_NAME"
+
+        # Account Key to use with the storage account
+        AccountKey = "AZURE_BLOB_ACCOUNT_KEY"
+
+        # Name of container in the blob storage
+        ContainerName = "AZURE_BLOB_CONTAINER_NAME"
+
+        Timeout = 300
```

Comment:
Remove this pls - we only have one top level timeout now.

Reply:
@scriptonist please remove the Timeout - there is only one global timeout now.

Reply:
bump